### DISCUSSION TITLE
Ollama Reject lobechat on different hosted setup.

### DISCUSSION THREAD

**Discussion by PumpkinB0Y** [lobehub/lobehub#2172] [Self Hosting]
Hi everyone, i have facing an issues that my ollama server hosted on my local machine, are rejecting request from lobechat (which setup at another server). Firewall are allowed for the port.

my lobechat Interface proxy address for ollama is : http://xx.xx.xx.xx:11434

For connectivity checking: 
`{
  "host": "xx.xx.xx.xx:11434",
  "message": "please check whether your ollama service is available",
  "provider": "ollama"
}`

Whereby in my Docker that run ollama have this log:
"[GIN] 2024/04/24 - 02:38:51 | 403 |       2.827µs |      172.17.0.1 | OPTIONS  "/api/tags"

Anyone have face this issues before? especially hosted Lobechat & ollama on different server.

**Comment by arvinxx** [comment]
Now you can try with `0.149.0`, I think it's fixed now.
